Atomic/backends/_docker.py

```diff
@@ -451,13 +464,20 @@ def check_args(cmd):
 
     def _running(self, con_obj, args, atomic):
         if con_obj.interactive:
-            cmd = [atomic.docker_binary(), "exec", "-t", "-i", con_obj.name, con_obj.command]
+            container_command = con_obj.command if not args.command else args.command
+            container_command = container_command if not isinstance(container_command, list) else " ".join(container_command)
+            cmd = [atomic.docker_binary(), "exec", "-t", "-i", con_obj.name, container_command]
             if args.display:
                 return atomic.display(cmd)
             else:
                 return util.check_call(cmd, stderr=DEVNULL)
         else:
-            cmd = [atomic.docker_binary(), "exec", "-t", "-i", con_obj.name] + con_obj.command
+            command = con_obj.command if not args.command else args.command
+            try:
+                cmd = [atomic.docker_binary(), "exec", "-t", "-i", con_obj.name] + command
+            except TypeError:
+                cmd = [atomic.docker_binary(), "exec", "-t", "-i", con_obj.name] + command
```

Comment:
This is the same line as before?  If it throws the first TypeError it will throw the second